src/test/java/reactor/kafka/util/ConsumerDelegateTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2021-2021 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.kafka.util;
+
+import org.junit.Test;
+
+/**
+ * @author Gary Russell
+ * @since 1.3.6
+ *
+ */
+public class ConsumerDelegateTests {
+
+    @SuppressWarnings("rawtypes")
+    @Test
+    public void testLag() {
+//        Consumer delegate = mock(Consumer.class);
+//        ConsumerDelegate consumer = new ConsumerDelegate(delegate);
+//        given(delegate.currentLag(any())).willReturn(OptionalLong.of(42L));
+//        assertEquals(42L, consumer.currentLag(new TopicPartition("foo", 0)).getAsLong());
```

Comment:
Are you going to finish this test?

Reply:
It requires the 3.0.0 client on the classpath - I used it to test with that client - commented out for older clients.